types/office-js/index.d.ts

```diff
@@ -19815,29 +19831,47 @@ declare namespace Office {
      */
     interface ReplyFormAttachment {
         /**
-         * Indicates the type of attachment. Must be file for a file attachment or item for an item attachment.
+         * Indicates the type of attachment.
+         *
+         * @remarks
+         *
+         * **Important**:
+         *
+         * - The `base64` attachment type was introduced in Mailbox requirement set 1.15. It's only supported by the `displayReplyAllForm`, `displayReplyAllFormAsync`, `displayReplyForm`, and
+         * `displayReplyFormAsync` methods.
+         *
+         * - The `cloud` attachment type isn't supported by the `displayNewMessageForm`, `displayNewMessageFormAsync`, `displayReplyAllForm`, `displayReplyAllFormAsync`,
+         * `displayReplyForm`, and `displayReplyFormAsync` methods.
          */
-        type: string;
+        type: MailboxEnums.AttachmentType | "base64" | "file" | "item";
```

Comment:
I noticed that this is a subset of the types available in the AttachmentType enum. Is there some significance to this?

Reply:
Good catch. We can just set the type as MailboxEnums.AttachmentType. I'll update this and other instances.